openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -817,3 +819,69 @@ def test_update_account_settings_rollback(self, mock_email_change):
         data = response.data
         self.assertEqual(old_email, data["email"])
         self.assertEqual(u"m", data["gender"])
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Account APIs are only supported in LMS')
+class TestAccountDeactivation(TestCase):
+    """
+    Tests the account deactivation endpoint.
+    """
+    PASSWORD = 'test'
+
+    def setUp(self):
+        super(TestAccountDeactivation, self).setUp()
+        self.superuser = SuperuserFactory(password=self.PASSWORD)
+        self.staff_user = AdminFactory(password=self.PASSWORD)
+        self.test_user = UserFactory()
+        self.url = reverse('accounts_deactivation', kwargs={'username': self.test_user.username})
```

Comment:
Are these paths not namespaced? :( 

No action is needed on your part. The person who initially implemented this should have used a namespace (e.g. 'user_api:v1:accounts:deactivate').